src/app/containers/PersonKort/PersonKort.tsx

```diff
@@ -0,0 +1,48 @@
+import {apiRoutes} from "../../../utils/http/apiConfig";
+import {LoadingIndicator} from "../../components/LoadingIndicator";
+import {PersonCard} from "@navikt/nap-person-card";
+import React from 'react';
+import styled from "styled-components";
+import useGet from "../../../utils/http/useGet";
+import {useTranslation} from "react-i18next";
+
+interface Props {
+  saksnummer: string;
+}
+
+const PersonKort: React.FunctionComponent<Props> = ({saksnummer}) => {
+  const {data, error, loading} = useGet(
+    `${apiRoutes().Overføringer}?status=Aktiv&saksnummer=${saksnummer}`,
+  );
+  const errorStatus = error?.response?.status;
+  const {t} = useTranslation();
+
+  return (
+    <>
+      {loading && <LoadingIndicator/>}
+      {data !== null &&
+      <PersonCardStyle>
+        <PersonCard name={`${t('personkort.saksnummer')}: ${data.saksnummer}`}
+                    fodselsnummer={`${t('personkort.fodselsnummer')}: ${data.identitetsnummer}`} gender={'unknown'}/>
```

Comment:
Det har neppe så mye å si akkurat her, men generelt er det en god vane å sende data inn som options i tekstfunksjonen i stedet for å "hardkode" teksten på denne måten. Altså; `name={t('personkort.saksnummer', {saksnummer})}`

For øvrig, hvis det blir for langt å ha alle props på samme linje, skriv dem på hver sin 🙂

Reply:
Veldig gott inspel! Har lagt till dette nå 👍